src/index.ts

```diff
@@ -993,6 +993,66 @@ function aionEncoder(data: Buffer): string {
   return '0x'.concat(data.toString('hex'));
 }
 
+/* tslint:disable:no-bitwise */
+function nulsAddressEncoder(data: Buffer): string {
+  const chainId = data[0] & 0xff | (data[1] & 0xff) << 8;
+  const tempBuffer = Buffer.allocUnsafe(data.length + 1);
+
+  let temp = 0;
+  let xor = 0x00;
+  for(let i = 0; i < data.length; i++) {
+    temp = data[i];
+    temp = temp > 127 ? temp - 256 : temp;
+    tempBuffer[i] = temp;
+    xor ^= temp;
+  }
+  tempBuffer[data.length] = xor;
+
+  let prefix = "";
+  if(1 === chainId) {
+    prefix = 'NULS';
+  } 
```

Comment:
Can you also support non production prefixes? 

https://github.com/nuls-io/nuls-v2-js-sdk/blob/cc208cdc153d3fca1e001fbe7c457780523bd96a/lib/api/sdk.js#L122

Reply:
Done.

Reply:
Can you also add non-production prefixes to the test?

Reply:
Done. Test cases are from: https://github.com/nuls-io/nuls-v2/blob/0d55b325438307319cc2bd385f4064c4518f705b/module/nuls-account/src/test/java/io/nuls/account/util/AddressToolTest.java